.github/workflows/test.yml

```diff
@@ -9,10 +9,10 @@ on:
 jobs:
   test:
     name: Ruby ${{ matrix.ruby }}
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby: [2.6, 2.7, 3.0]
+        ruby: [2.6, 2.7, '3.0', 3.1]
```

Comment:
3.0 has to be quoted because of this issue https://github.com/actions/runner/issues/849.